src/platform-includes/performance/configure-sample-rate/javascript.mdx

```diff
@@ -3,18 +3,14 @@
 // substitute its name for `@sentry/browser` here
 import * as Sentry from "@sentry/browser";
 
-// If taking advantage of automatic instrumentation (highly recommended)
-import { BrowserTracing } from "@sentry/tracing";
-// Or, if only manually tracing
-// import * as _ from "@sentry/tracing"
-// Note: You MUST import the package in some way for tracing to work
-
 Sentry.init({
   dsn: "___PUBLIC_DSN___",
 
   // This enables automatic instrumentation (highly recommended), but is not
-  // necessary for purely manual usage
-  integrations: [new BrowserTracing()],
+  // necessary for purely manual usage.
+  // If you only want to use manually, remove the BrowserTracing integration and add
```

Comment:
l: I think a word is missing here. Maybe something along these lines:
(likewise for the framework includes below)
```suggestion
  // If you only want to use manual instrumentation, remove the BrowserTracing integration and add
```